src/site/_includes/components/component-preview/macro.njk

```diff
@@ -49,6 +54,32 @@
 {% endcall -%}
 {% endset -%}
 
+{% set tabs = [{
+  label: "HTML",
+  contents: htmlTabContents | safe,
+  panelId: idPrefix + "--html"
+}, {
+  label: "Nunjucks",
+  contents: nunjucksTabContents | safe,
+  panelId: idPrefix + "--nunjucks"
+}] -%}
+
+{% if reactExampleCode -%}
+
+{% set reactTabContents %}
+{% call guidanceCodeSnippet({ language: "tsx" }) -%}
+{{ reactExampleCode | safe -}}
+{% endcall -%}
+{% endset -%}
+
+{% set tabs = (tabs.push({
+  label: "React",
+  contents: reactTabContents | safe,
+  panelId: idPrefix + "--tsx"
+}), tabs) -%}
```

Comment:
Awkward way of appending to an array in Nunjucks (alternative is to define a custom filter with a nicer syntax...)

Reply:
I think this is ok because the feature flag is only a temporary measure

Reply:
Will be needed as long as some components don't have React versions though